main.go

```diff
@@ -278,10 +278,10 @@ func outputToConsole(issues chan *Issue) int {
 	return status
 }
 
-func outputToJSON(issues chan *Issue) int {
+func outputToJSON(chIssues chan *Issue) int {
```

Comment:
What's the purpose of all these issues -> chIssues renames?

Reply:
When `issues` was a separate package the variable name was masking the package name, so i had to change the variable. Now that the package is gone I could revert these.

Reply:
Reverted these changes